CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ option(VIKUNJA_ENABLE_EXTRA_WARNINGS "Enable extra warnings" OFF)
 option(BUILD_TESTING "Build the testing tree." OFF)
 cmake_dependent_option(VIKUNJA_SYSTEM_CATCH2 "Use your local installation of Catch2" ON BUILD_TESTING OFF)
```

Comment:
There is a design decision to be made: The default will download Catch2 -- this can be considered intrusive.
On the other hand download by default makes it simpler for naive users. Your call.

Reply:
Good point. I opened an issue #73  and will fix it in another PR.